packages/typespec-vscode/README.md

```diff
@@ -1,25 +1,74 @@
-# TypeSpec Language Support for VS Code
+# TypeSpec Extension in Visual Studio Code
 
 [Repository](https://github.com/microsoft/typespec) | [Documentation](https://typespec.io/docs) | [Issues](https://github.com/microsoft/typespec/issues) | [Samples](https://github.com/microsoft/typespec/tree/main/packages/samples/specs)
 
-This provides provides [TypeSpec](https://github.com/microsoft/typespec) language support for VS Code.
+The TypeSpec extension for Visual Studio Code enhances the development of TypeSpec by leveraging VS Code's powerful features. It provides:
 
-![](https://raw.githubusercontent.com/microsoft/typespec/main/docs/images/vscode.gif)
+- IntelliSense and syntax highlighting
+- Code autocompletion and formatting
+- Live diagnostics and quick fixes
+- Refactoring tools (rename, go-to definition, etc.)
+- Seamless project setup and emitter configuration
```

Comment:
shall we mention `Generate code from typespec` feature here?

Reply:
Added